src/kirby/components/list/list.component.ts

```diff
@@ -24,7 +24,9 @@ import { GroupByPipe } from './pipes/group-by.pipe';
 import { ListSwipeAction } from './list-swipe-action';
 import { ThemeColor } from '@kirbydesign/designsystem/helpers/theme-color.type';
 
-export type ListShape = 'square' | 'rounded';
+export type ListShape = 'square' | 'rounded' | 'none';
+
+export type ItemSpacing = 'true' | 'false';
```

Comment:
What is the purpose of making `ItemSpacing` a string based type over a boolean?

Reply:
Well.. 
1: hasItemSpacing="false" is resulting in a string value, i.e.: 'false' is truthy.
And 2: We hope to extend this later, with values such as "compact" or "spacey"

Reply:
1. I actually didn't know. Well today i learned.
2. Fine.

Reply:
We don't do this in any other components, so I'm reverting to boolean and closing this conversation